python/WlvHbb13TeV2016config/volatile.ini

```diff
@@ -1,5 +1,5 @@
 # this file has been created automatically and will be overwritten by submit.py!
 [Directories]
-plotpath: /mnt/t3nfs01/data01/shome/gaperrin/VHbb/CMSSW_7_4_3/src/Xbb/python/logs_v25_Wlv//sysnew-TEST5/Plots/
-logpath: /mnt/t3nfs01/data01/shome/gaperrin/VHbb/CMSSW_7_4_3/src/Xbb/python/logs_v25_Wlv//sysnew-TEST5/Logs/
```

Comment:
this should not be committed, I think

Reply:
This is ok (the path is overwritten at each job submission)

Reply:
the file itself should not be committed, since it will always have changes, that's why the automatically generated lines are in a separate file

Reply:
sorry I didn't notice it was the volatile.ini file. It is now removed.